formfactory/fields.py

```diff
@@ -0,0 +1,30 @@
+import markdown
+import django
```

Comment:
Group the django import with the other django imports.

Reply:
Removed, a stale import.